src/pages/Subscriptions.tsx

```diff
@@ -725,7 +811,27 @@ export const Subscriptions = ({ signOut }: { signOut: () => void }) => {
 
   return (
     <Box sx={{ overflow: "auto", height: "100%" }}>
-      {renderSubscriptionPage()}
+      {activeSubscriptions?.length > 0 && !showConfirmPage ? (
+        <Box>
+          <BackButton route="/profile" />
+          <Box sx={{ padding: "1rem" }}>
+            <Heading>Subscriptions</Heading>
+            {activeSubscriptions.map((subscription, index) => (
+              <Box sx={{
+                borderRadius: "4px",
+                marginTop: "2rem",
+                padding: "1rem",
+                boxShadow: "rgb(2 2 40 / 8%) 0px 4px 6px",
+              }}>
+                <SubscriptionCard subscription={subscription} key={index}/>
+              </Box>
+            ))}
+            <Typography variant="body2" sx={{ paddingTop: "1.5rem", textAlign: "center" }}>
+              To manage your subscriptions, please reach out to us via chat.
+            </Typography>
+          </Box>
+        </Box>
+      ) : renderSubscriptionPage()}
```

Comment:
```suggestion
      ) :  <SubscriptionPage /> }
```
calling a function component is not recommended -> https://kentcdodds.com/blog/dont-call-a-react-function-component

Reply:
Youre right sorry this was test code. i will address this in a feature pr!